### ISSUE TITLE
Parallel Initial Install

### ISSUE THREAD

**Issue by jjzmajic** [zdharma/zinit#288]
Now that updates have been parallelized, wouldn't parallelized cloning (at first install) be a good default (like plug or vimplug0. Sometimes I'm too tired to debug an issue so I just nuke the ~/.zinit directory. I'm sure I'm not the only one.

**Comment by psprint** [comment]
That's true, such approach would be a good idea. However, it is a little difficult to achieve – the install should be done within a subshell, while the following loading within the current shell. I would need a $10-15 to motivate myself to sit to this…

**Comment by jjzmajic** [comment]
Fair enough, you earned it! Hope it find you well :+1:

**Comment by jjzmajic** [comment]
Hi @psprint, any updates on this issue?

**Comment by psprint** [comment]
Ah, sorry, I've missed it. I'll find time for it soon.

**Comment by jjzmajic** [comment]
Gentle bump :smile:

Hope everything is going alright on your end. I know a lot of stuff got derailed with COVID so I completely understand if you haven't found the time.
